docs/book/releases/v2.0.0.md

```diff
@@ -80,6 +80,14 @@
 10. `user` and `ca-file` change in the [vsphere-config-secret](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/installation.html#create-a-configuration-file-with-vsphere-credentials-) is not honored until vSphere CSI Controller Pod restarts.
     - Impact: Volume lifecycle operations fail until the controller pod restarts.
     - Workaround: Restart `vsphere-csi-controller` deployment Pod.
+11. vCenter restart may leave Kubernetes persistent volume claims in the Pending state with error message - `failed to get taskInfo for CreateVolume task from vCenter "vCenter IP" with err: ServerFaultCode: The session is not authenticated`.
+    - Impact:
+        - Persistent volume claims which are being created at the time vCenter is restarting may remain in the `pending` state for 1 hour.
+        - After vSphere CSI Driver clears up pending cached tasks, new tasks to create volumes will be issued to the vCenter and then Persistent volume claims can go into `Bound` State.
+        - Restarting vCenter while volumes are getting created may leave orphan volumes on the datastores.
+    - Workaround:
+        - Make sure vCenter is restarted in the maintenance window when Kubernetes users are not actively provisioning new volumes.
```

Comment:
I dont think the VI admin can plan this way. Its hard to know when the apps team(DevOps) is going to use k8s clusters. Having said that, I do not mind mentioning this as an option :)

Reply:
Removed this from the workaround.